SPECS/hyperv-daemons/hyperv-daemons.spec

```diff
@@ -19,6 +19,7 @@ Source0:        https://github.com/microsoft/WSL2-Linux-Kernel/archive/linux-msf
 # HYPERV KVP DAEMON
 Source1:        hypervkvpd.service
 Source2:        hypervkvp.rules
+Source3:        hv_set_ifconfig
```

Comment:
Is that expected that there is no .sh extension?

Reply:
Yes. The actual target file is hv_set_ifconfig